python/tk_multi_publish2/summary_overlay.py

```diff
@@ -63,7 +63,7 @@ def show_fail(self):
             QtGui.QPixmap(":/tk_multi_publish2/publish_failed.png")
         )
         self.ui.label.setText("Publish\nFailed!")
-        self.ui.details.setText("Please see error log for more details")
+        self.ui.details.setText("Click here for more details")
```

Comment:
While we're in here, let's just make the text consistent for success/fail. `"For more details, <b>click here</b>."` would be good I think.